doc/source/ray-core/package-ref.rst

```diff
@@ -239,6 +239,10 @@ Debugging APIs
 The Ray Command Line API
 ------------------------
 
+Cluster Management
```

Comment:
I thought we are going to group commands printed by `ray --help`?

Reply:
Yeah, you should be able to define command groups there in click I believe.

Reply:
I did some research, and it is a bit difficult (there's no public API to customize help messages) https://stackoverflow.com/questions/62182687/custom-help-in-python-click

Reply:
Should we do this in this PR? I am planning to cherry pick for 2.0, but this change might be a bit risky